pages/common/dict.md

```diff
@@ -0,0 +1,24 @@
+# dict
+
+> Client/server software, human language dictionary databases, and tools supporting the DICT protocol.
```

Comment:
I think that description is very vague and dosn't really tell what the command is about. I know that's what the official page uses, but could we maybe make this a little more concrete?

Reply:
I looked up dictd but not really grasp all of its feature, I was afraid making short definition wouldn't fully cover or make the wrong assumption about the dictd package itself. But for the dictionary feature, it's really useful.